src/Module/Item/Ignored.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Friendica\Module\Item;
+
+use Friendica\App;
+use Friendica\BaseModule;
+use Friendica\Core\L10n\L10n;
+use Friendica\Core\Session;
+use Friendica\Core\System;
+use Friendica\Database\Database;
+use Friendica\Model\Item;
+use Friendica\Network\HTTPException;
+
+/**
+ * Module for ignoring threads or user items
+ */
+class Ignored extends BaseModule
```

Comment:
Sorry, I forgot to ask to rename this module to `Ignore`.

Reply:
Wouldn't it be consequent to rename the endpoint to ```/ignore``` as well?

Reply:
yep .. Thought so too ..